#!/usr/bin/env python
PACKAGE = "pole_based_localization"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("use_gps", bool_t, 0, "Toggle use of gps sensor", True)
gen.add("max_positional_correction", double_t, 0, "max applied positional correction per step [m]", 0.2, 0.0, 1.0)
gen.add("max_angular_correction", double_t, 0, "max applied angular correction per step [rad]", 0.0002, 0.0, 0.05)
gen.add("pose_correction_dampening", double_t, 0, "dampening of cached positional correction [%]", 2.5, 0, 10)

exit(gen.generate(PACKAGE, "pole_based_localization", "pole_position_correction"))

